### PR TITLE
Add properties to table head if defined

### PR DESCRIPTION
**Pull request by richardhj**
## Description

I want to propose the following change:
"Add the properties to the table head if defined."

Reason:
I add custom properties via the `PopulateEnvironmentEvent`.
```php
       /** @var Contao2BackendViewDefinitionInterface $viewSection */
        $viewSection = $definition->getDefinition(Contao2BackendViewDefinitionInterface::NAME);
        $formatter   = $viewSection->getListingConfig()->getLabelFormatter($definition->getName());

        $propertyNames   = $formatter->getPropertyNames();
        $propertyNames[] = 'paid_status';
```

Then, the table looks like this (header `td` is missing):
Screen Shot 2019-05-31 at 12 42 44

With my change, the table is correct:
Screen Shot 2019-05-31 at 12 40 16

## Checklist
- [x] Read and understood the [CONTRIBUTING guidelines](CONTRIBUTING.md)
- [x] Created tests, if possible
- [x] All tests passing
- [x] Extended the README / documentation, if necessary
- [x] Added myself to the `@authors` in touched PHP files
- [x] Checked the changes with phpcq and introduced no new issues

### CODE REVIEW

File: src/Contao/View/Contao2BackendView/ActionHandler/AbstractListShowAllHandler.php [hunk 1]
```diff
@@ -480,15 +480,20 @@ private function getTableHead(EnvironmentInterface $environment)
         $sorting    = ViewHelpers::getCurrentSorting($environment);
         $columns    = $this->getSortingColumns($sorting);
         foreach ($formatter->getPropertyNames() as $field) {
-            // Skip unknown properties. This may happen if the property is not defined for editing but only listing.
-            if (!$properties->hasProperty($field)) {
-                continue;
+            if ($properties->hasProperty($field)) {
```
Comment (discordier) [inline src/Contao/View/Contao2BackendView/ActionHandler/AbstractListShowAllHandler.php]: I would rephrase this to keep the original implementation below but switching only the `'content'` part.
Like:
```php
$tableHead[] = [
    'class'   => 'tl_folder_tlist col_' . $field . (\in_array($field, $columns) ? ' ordered_by' : ''),
    'content' => $properties->hasProperty($field)) 
        ? $properties->getProperty($field)->getLabel()
        : $this->translate($definition->getName() . '.' . $field . '.0', 'contao_' . $definition->getName())
];
```
IMO the implementation is clearer then.

**Comment by baumannsven** [comment]
Merged in 99c153d48ac4749f89de2481b11e43de9770f92c
